src/main/resources/update_user_table.sql

```diff
@@ -0,0 +1,2 @@
+alter table user drop if exists username;
```

Comment:
Why we need this script?

Reply:
I think we don't. While I was doing some research about the best way to change Username to Email for Login request I found this video and adapted the solution to our proyect.
https://www.youtube.com/watch?v=IcDDQ2qZJ-s&t=206s
Since we dont have username attr in the User entity, this script is usseless!